src/toil/test/provisioners/clusterScalerTest.py

```diff
@@ -354,17 +342,20 @@ def stop(self):
                     return time.time() - self.startTime
 
             for i in xrange(numNodes):
```

Comment:
Oops, you're totally right, good catch. Sadly this got merged already--let me see if github handles pushing to the same branch or if fixing that would require a new PR.